Makefile

```diff
@@ -16,4 +16,5 @@ test: image
 citest:
 	docker run --rm \
 		--workdir /usr/src/app \
+		--volume /tmp:/tmp \
```

Comment:
this seems fine, but I'm curious what motivated this. (I always appreciate more context in commit messages for this kind of thing)